modules/tracing/mule-tracer-exporter-impl/pom.xml

```diff
@@ -89,7 +85,11 @@
                 </exclusion>
             </exclusions>
         </dependency>
-        <!-- Added back because this was brought transitively by okio -->
+        <dependency>
```

Comment:
why is this needed?

Reply:
Which dependency do you mean?

Reply:
Now you don't have the exclusion opentelemetry-exporter-otlp-common but in opentelemetry-exporter-otlp. Is that correct?

Reply:
Yes, on the new opentelemetry version, okio comes with `opentelemetry-exporter-otlp` instead of `opentelemetry-exporter-otlp-common` so I moved the exclusion from `opentelemetry-exporter-otlp-common` to `opentelemetry-exporter-otlp`